content/plugins/commons-chunk-plugin.md

```diff
@@ -149,9 +150,13 @@ new webpack.optimize.CommonsChunkPlugin({
 
 You also have the ability to pass the `minChunks` property a function. This function is called by the `CommonsChunkPlugin` and calls the function with `module` and `count` arguments.
 
-The `module` property represents each module in the chunks you have provided via the `names` property.
+The `module` argument represents each module in the chunks you have provided via the `name`/`names` property.
+`module` has the shape of a [NormalModule](https://github.com/webpack/webpack/blob/master/lib/NormalModule.js#L57), which has two particularly useful properties for this use case:
```

Comment:
Linking to a specific line on master might break on a new commit. Either drop `#L57` or s/master/70e48d2/.